architecture/openshift_model.adoc

```diff
@@ -317,9 +320,7 @@ Sample Template:
 ----
 
 == Route
-An OpenShift route is a way to announce your service to the world.  A route, consumed by a router in conjunction with
- service endpoints provides named connectivity from external sources to your applications.  Each route provides a name, service
- selector, and optionally security configuration.
+An OpenShift route is a way to announce your service to the world.  A route, consumed by a router in conjunction with service endpoints provides named connectivity from external sources to your applications.  Each route provides a name, service selector, and optionally security configuration.
```

Comment:
Maybe add a comma: "A route, consumed by a router in conjunction with service endpoints, provides named connectivity from external sources to your applications.